frontend/src/app/globe-ngb/globe-ngb-testing.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
+import { NgbConfig } from '@ng-bootstrap/ng-bootstrap';
+import { GlobeNgbModule } from './globe-ngb.module';
+
+/**
+ * A module for unit tests, which imports the root GlobeNgbModule and disables
+ * animations
+ */
+@NgModule({
+  imports: [GlobeNgbModule.forRoot()],
+  exports: [GlobeNgbModule]
+})
+export class GlobeNgbTestingModule {
+  constructor(ngbConfig: NgbConfig) {
+    ngbConfig.animation = false;
```

Comment:
OOC: is it necessary? I read that the animations were pure CSS, so that should not impact existing tests right?

Reply:
It is for two main reasons:

- the tests for ngbCollapse assume that the transition is synchronous (they test that the `show` class is present or absent)
- the tests for the confirm service and modal assume that the transition is synchronous (they test that the modal content is removed from the DOM, which happens only at the end of the transition)